winter/messaging/messaging_config.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+from typing import Dict
+from typing import Set
+from typing import Set
+
+
+@dataclass
+class MessagingConfig:
+    topics: Set[str]
```

Comment:
@mikhaillazko Why do we need to specify topic? I mean, when we want to start `consumer_worker`, we run

`python run consumer_worker consumer_name`
Then we get all topics from events, which were discovered in packages, that specified for this consumers.

Why do we need such a check?
```
declared_topics = self._config.topics
        if declared_topics != discovered_topics:
            raise InvalidTopologyException(f'Not all topics are declared: {discovered_topics - declared_topics}')
```

https://github.com/WinterFramework/winter/pull/247/files#diff-405b2970ce55f50c07123f61d53ad1046c7d033749bde568090520371a9bdd22R68

Reply:
answered it below. it's just more safer when you can validate that configuration matched with code annotations